lib/index.js

```diff
@@ -1,6 +1,7 @@
 const fetch = require("node-fetch");
 const colors = require("colors");
 const toColors = require("hex-to-color-name");
+var headers = {}
```

Comment:
No magic globals please, let's make the client ID an argument to the relevant functions.